src/guiguts/application.py

```diff
@@ -353,6 +353,33 @@ def init_edit_menu(self) -> None:
         menu_edit.add_button(
             "Colu~mn Paste", maintext().columnize_paste, "Cmd/Ctrl+Shift+V"
         )
+        menu_edit.add_separator()
+        # NOTE: this was "~l" in GG1; but "~l" is now used by Column Cut, using "~w"
+        menu_edit.add_button(
```

Comment:
To avoid large menus, we could consider putting these in a submenu. I'm not sure of the balance between large menus and deep menus, so happy to go with whatever you think.

Reply:
I briefly wondered about this myself, and the answer might depend on how closely we are planning to hew to the shape of GG1. I placed these in the same place I found them in GG1, but maybe we plan to make a new shape? Have you thought through that aspect already?

Reply:
That's a really tricky question, and one I'm trying to put off a little longer, as far as it's possible to continue without making lots of extra work, either to match GG1 now, or the change to match in future. I'd imagined that once there were a few features in GG2, but well before it's actually a fully useful tool, we'd make a version available and engage existing PPers in a discussion about it. Closeness to GG1 would likely be a significant part of that. That said, I'm very happy to hear comments/suggestions about it at any point.

Reply:
I support the "put it off" approach. If we mimic the GG1 menus as we add things, we don't have to think about it much. We can overhaul the menu structure later on when the app has a more mature shape. Restructuring the menus seems easy from the code I've seen so far...